go.mod

```diff
@@ -5,7 +5,7 @@ require (
 	github.com/NebulousLabs/fastrand v0.0.0-20181203155948-6fb6489aac4e // indirect
 	github.com/NebulousLabs/go-upnp v0.0.0-20180202185039-29b680b06c82
 	github.com/Yawning/aez v0.0.0-20180114000226-4dad034d9db2
-	github.com/btcsuite/btcd v0.20.0-beta
+	github.com/btcsuite/btcd v0.20.0-beta.0.20191109022137-037489ef505e
```

Comment:
Should update:  
  * `btcd` to `v0.20.1`
  * `neutrino` to `v0.11.0`
  * `btcwallet` to `v0.11.0`

Reply:
Done.